src/discordcr/mappings/oauth2.cr

```diff
@@ -0,0 +1,20 @@
+require "./converters"
+require "./user"
+
+module Discord
+  # An Oauth2 application, as registered with Discord, that can hold
+  # information about a `Client`'s associated bot user account and owner,
+  # among other Oauth2 properties.
+  struct Oauth2Application
```

Comment:
Shouldn't this be `OAuth2Application`? I'm not 100% familiar with the naming standards (@RX14?)